packages/fxa-shared/subscriptions/types.ts

```diff
@@ -165,9 +165,12 @@ export type WebSubscription = Pick<
       'incomplete' | 'incomplete_expired'
     >;
     subscription_id: Stripe.Subscription['id'];
+    promotion_amount_off?: string | null;
     promotion_code?: string;
```

Comment:
Unrelated, but i have cursed eyes... Should this also accept `null`, given this code above:
```js
  promotion_code: isA
    .string()
    .optional()
    .allow(null)
    .description(DESCRIPTIONS.promotionCode),
```
(unless the `null` is casting to an empty string somewhere in the pipeline, in which case ignore this comment.)